lightbeam

```diff
@@ -0,0 +1 @@
+/home/jef/Documents/GitHub/Vurich/lightbeam
```

Comment:
It looks like your submodule is pointing to a local directory. Should likely be pointed at the git repo.